.github/workflows/build.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build and test operator images on push to every branch except main and pull requests
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches-ignore: [main]
+jobs:
+  build-test-operator:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+            ~/go/bin/
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - uses: actions/setup-go@v2
+        with:
```

Comment:
Is this necessary? Can you just use default, and if something is backwards compat we can fix it when the action breaks?

Reply:
Just specifying the version.